contracts/event/LemonadeEventAttestation.sol

```diff
@@ -0,0 +1,179 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.19;
+
+import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@ethereum-attestation-service/eas-contracts/contracts/IEAS.sol";
+import "@ethereum-attestation-service/eas-contracts/contracts/resolver/SchemaResolver.sol";
+import "@ethereum-attestation-service/eas-contracts/contracts/resolver/examples/AttesterResolver.sol";
+
+import "./ILemonadeEventAttestation.sol";
+import "./resolvers/EventHostSchemaResolver.sol";
+import "./resolvers/TicketIssuingSchemaResolver.sol";
+
+contract Event {}
```

Comment:
What is this used for?

Reply:
@ChrisLahaye I would need to create an unique address every time user registers an event. I can see that we can obtain this by instantiating a new instance of any contract (hence I create an empty `Event` contract for this purpose).